ui/packages/consul-ui/tests/pages/dc/intentions/edit.js

```diff
@@ -10,7 +10,12 @@ export default function(
 ) {
   return {
     scope: 'main',
-    visit: visitable(['/:dc/intentions/:intention', '/:dc/intentions/create']),
+    visit: visitable([
+      '/:dc/intentions/:intention',
+      '/:dc/services/:service/intentions/:intention',
+      '/:dc/services/:service/intentions/create',
+      '/:dc/intentions/create',
+    ]),
```

Comment:
In the acceptance tests when we write

When I visit the _intention_ page for yaml:

Depending on the data you pass to that step, it will try each of these paths/urls in order until it finds one where it has all the data for that path - so in this case if I don't pass a service slug it will fall through to the last entry here.